### DISCUSSION TITLE
Chrome does not store the authority certificate

### DISCUSSION THREAD

**Discussion by twilwa** [coddingtonbear/obsidian-local-rest-api#131] [Q&A]
Little bit of a quirk and didn't used to happen, I don't think -- I have to re-import the generated cert on any new chrome session. Lmk if there's anything I should test or logs I should grab :)

**Comment by coddingtonbear** [comment]
Unfortunately, although this browser extension runs on Chrome, I'm not a Chrome maintainer.  If you reach out on https://support.google.com/chrome/community?hl=en they may be able to point you in the right direction!  Good luck!

**Comment by coddingtonbear** [comment]
I've converted your issue to a discussion since it might be that somebody else has experienced something similar and might be able to help you work around the issue.  For what it's worth, certificate management varies based on not only browser but also operating system. 

Although I can personally confirm that I don't experience the same problem you're running on the latest version of Chrome on Linux, there are, of course, other versions of Chrome, other browsers that people might call Chrome (but are actually derivatives of Chromium), and other operating systems.  You'll have better luck if you clarify exactly what OS you're using and what version of exactly what browser.
